### DISCUSSION TITLE
Video stutter on recordings

### DISCUSSION THREAD

**Discussion by peter247** [blakeblackshear/frigate#2456] [General]
Trying to fix a video stutter on the recordings, I have a single rlc-510 camera and using CPU detection for a coral usb device is almost impossible to get without paying 2-3 time it's value.
I'm using a ryzen 5700g and installed frigate on bare metal of proxmox so I can get to the hardware video acceleration etc . 
I drop the frame rate of the camera to 25 fps and using the lowest resolution the camera will go 2304x1296 , ffmpeg says about 15% of a  core and the object detection is not making my system work , but every about 2 seconds I get a stutter on the recordings , the live and debug look o.k , but it is hard to tell at 5fps .

my config file , I did a change this --
    detect:                 
      width: 2304
      height: 1296
      fps: 5

[Copy.txt](https://github.com/blakeblackshear/frigate/files/7726969/Copy.txt)

**Comment by blakeblackshear** [comment]
https://docs.frigate.video/configuration/camera_specific#reolink-410520-possibly-others

**Reply by peter247** [reply]
tried that I just get a green screen , rebooted the camera and got a picture , after deleting  rw_timeout.
ERROR   : Option rw_timeout not found.

**Reply by blakeblackshear** [reply]
That probably means you didn't change the path to something other than rtsp. Can you open a proper issue? Discussions aren't meant for these kinds of support issues.

**Reply by peter247** [reply]
The ERROR : Option rw_timeout not found , as been raised before in support .

BUT ... I think I've found my stuttering problem and I think the problem is just with my docker have a melt down with me setting the network to bridged when I had 2 network card on 2 sub nets.
No stuttering , but new error with what I've added to fix the first problem.
